storage/engine/rocksdb_native/rocksdb_native.go

```diff
@@ -15,7 +15,26 @@
 //
 // Author: Peter Mattis (peter@cockroachlabs.com)
 
-package engine
+package rocksdb_native
 
-// #cgo LDFLAGS: -lrt
+import (
+	_ "github.com/cockroachdb/c-protobuf"
+	_ "github.com/cockroachdb/c-rocksdb"
+	_ "github.com/cockroachdb/c-snappy"
+
+	"github.com/cockroachdb/cockroach/util/log"
+)
+
+// #cgo CPPFLAGS: -I ../../../../c-protobuf/internal/src -I ../../../../c-rocksdb/internal/include
+// #cgo CXXFLAGS: -std=c++11
+// #cgo darwin LDFLAGS: -Wl,-undefined -Wl,dynamic_lookup
+// #cgo !darwin LDFLAGS: -Wl,-unresolved-symbols=ignore-all
+// #cgo linux LDFLAGS: -lrt
 import "C"
+
+//export rocksDBLog
+func rocksDBLog(s *C.char, n C.int) {
+	// Note that rocksdb logging is only enabled if log.V(1) is true
+	// when RocksDB.Open() is called.
+	log.Infof("%s", C.GoStringN(s, n))
```

Comment:
The dependency on `util/log` is unfortunate. You'll still rebuild `rocksdb_native` if `util/log` or any of its dependencies change. Might be worth adding something like:

```
var Logger = func(string, ...interface{}) {}
```

Then `storage/engine` can set `rocksdb_native.Logger = log.Infof`.